tests/fsharpqa/Source/EntryPoint/W_NoEntryPointTypeInNamespace.fs

```diff
@@ -2,7 +2,7 @@
 // Empty program entry point warning
 // Lack of entry point produces warning with correct source location when compiled to *.exe
 // when source file declares type inside namespace
-//<Expects id="FS0988" span="(1,81-1,81)" status="warning">Main module of program is empty: nothing will happen when it is run</Expects>
+//<Expects id="FS0988" span="(10,18-10,18)" status="warning">Main module of program is empty: nothing will happen when it is run</Expects>
```

Comment:
it's at https://github.com/Microsoft/visualfsharp/blob/master/src/fsharp/IlxGen.fs#L5912